### PR TITLE
Features/create docker container 176775180

### PR DESCRIPTION
**Pull request by aneyzberg**
Problem
=======
We would like to be able to spin up a node that connects to our testnet
[link to Pivotal Tracker #176775180)](https://www.pivotaltracker.com/story/show/176775180)

Solution
========
Spin up a docker node that starts an openethereum instance with the correct configuration to connect to our testnet.

Change summary:
---------------
* Add Dockerfile
* Add entrypoint script that writes validators to `liberty-testnet-spec.json` file
* Add .env.sample file
* Update README

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,3 +1,6 @@
 TESTNET_ACCOUNT_PRIVATE_KEY=private key for an account we have on testnet that has eth.
 TESTNET_CHAIN_URL=url to connect to testnet
 LOCAL_NETWORK_ACCOUNT_PRIVATE_KEY=private key for an account we have on our local network that has eth.
+VALIDATOR1=address for validator running on node called liberty-chain
+VALIDATOR2=address for validator running on node called liberty-chain1
+BOOTNODE=enode address of node this node would like to sync with
```
Comment (wilwade) [inline .env.sample:6]: Are these 0x prefixed addresses or does it not matter?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM openethereum/openethereum:v3.1.1
+
+USER root
+RUN apk update && apk add bash
+RUN apk add --no-cache jq
```
Comment (wilwade) [inline Dockerfile:5]: I'm not sure we want bash in the image, but if so:

```suggestion
RUN apk update
RUN apk add --no-cache jq bash
```

File: docker/liberty-testnet-spec.json [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        "name": "identity",
+        "pricing": {
+          "linear": {
+            "base": 15,
+            "word": 3
+          }
+        }
+      }
+    },
+    "0x20f2478FecAbD3D9aA7e13e1D961ED53c9161574": { "balance": "10000000000000000000000" },
```
Comment (wilwade) [inline docker/liberty-testnet-spec.json:104]: I'm curious, do we need this part of the spec here? The other bits are important, but I don't think this is.

File: .env.sample [hunk 4]
```diff
@@ -1,3 +1,6 @@
 TESTNET_ACCOUNT_PRIVATE_KEY=private key for an account we have on testnet that has eth.
 TESTNET_CHAIN_URL=url to connect to testnet
 LOCAL_NETWORK_ACCOUNT_PRIVATE_KEY=private key for an account we have on our local network that has eth.
+VALIDATOR1=address for validator running on node called liberty-chain
```
Comment (wilwade) [inline .env.sample:4]: Do we want to have these in the same file as the env for other purposes? I wonder if these should just be in the docker directory (and just make sure the copy command doesn't include it through ignore or individual file inclusion)

File: README.md [hunk 5]
```diff
@@ -22,7 +22,10 @@ Environment Variables
 | LOCAL_NETWORK_PRIVATE_KEY         | private key for an account we have on our local network that has eth.         | 
 | TESTNET_ACCOUNT_PRIVATE_KEY       | private key for an account we have on our test network that has eth.         |
 | TESTNET_CHAIN_URL     | Url to connect to tesnet        |     
-  
+| VALIDATOR1 | Public key of validator running on liberty-chain |
+| VALIDATOR2 | Public key of validator running on Liberty chain
+| BOOTNODE | enode address of the node our node would like to connect to |
+
```
Comment (wilwade) [inline README.md:28]: Do we want a README in the docker directory instead of adding these here?

**Review by wilwade — CHANGES_REQUESTED** [review]
Few questions and comments

**Comment by enddynayn** [comment]
👍

File: .env.sample (hunk 1 above)
Comment (aneyzberg) [reply .env.sample:6]: it does not matter but I can specify.

File: docker/liberty-testnet-spec.json (hunk 3 above)
Comment (aneyzberg) [reply docker/liberty-testnet-spec.json:104]: yeah this is a good point. I supposed we do not need this.

**Review by wilwade — APPROVED** [review]

**Pull request merged**